components/chainhook-sdk/src/chainhooks/types.rs

```diff
@@ -380,6 +406,7 @@ impl StacksChainhookFullSpecification {
             predicate: spec.predicate,
             action: spec.action,
             enabled: false,
+            expired: false,
```

Comment:
Is the state `enabled = true, expired = true` reachable?

Reply:
No, it isn't. Do you want to use just `enabled`?

Reply:
I am not sure, it feels like that `expired` is redundant, but we need some information.
Could 
```rust
/// Block height
expired_at: Option<u64>
```
be useful?

Reply:
Changed: https://github.com/hirosystems/chainhook/pull/397/commits/f6970c7075cd623041ff348f74213abacfc813c5